### PR TITLE
Document workaround for jackson serialiaztion of scala case objects

### PR DESCRIPTION
**Pull request by octonato**

### CODE REVIEW

File: akka-docs/src/main/paradox/serialization-jackson.md [hunk 1]
```diff
@@ -164,11 +164,20 @@ when using polymorphic types.
 
 ### ADT with trait and case object
 
-In Scala it's common to use a sealed trait and case objects to represent enums. If the values are case classes
+In Scala, it's common to use a sealed trait and case objects to represent enums. If the values are case classes
```
Comment (johanandren) [inline akka-docs/src/main/paradox/serialization-jackson.md:167]: This comma sounds more weird to me than the original sentence?

**Review by johanandren — APPROVED** [review]
LGTM with one comment on grammar

**Review by ennru — APPROVED** [review]
LGTM. (The comma shouldn't be added.)

**Review by patriknw — APPROVED** [review]
LGTM

File: akka-docs/src/main/paradox/serialization-jackson.md (hunk 1 above)
Comment (renatocaval) [reply akka-docs/src/main/paradox/serialization-jackson.md:167]: I just followed the grammar suggestion from IntelliJ

CleanShot 2020-09-01 at 12 53 05

Comment (renatocaval) [reply akka-docs/src/main/paradox/serialization-jackson.md:167]: the name of the grammar rule is: "Missing comma after an introductory phase"

Comment (ennru) [reply akka-docs/src/main/paradox/serialization-jackson.md:167]: This is not an introductory phrase. Only a bit unusual sentence order: "It is common in Scala to ..."

Comment (renatocaval) [inline akka-docs/src/main/paradox/serialization-jackson.md:167]:
```suggestion
It's common in Scala to use a sealed trait and case objects to represent enums. If the values are case classes
```

**Pull request merged**